### PR TITLE
Fixing Padding/Margin

### PR DESCRIPTION
**Pull request by wylie**
Adding some padding and negative margin so the grid doesn't add more padding

### CODE REVIEW

File: less/_grid.less [hunk 1]
```diff
@@ -102,7 +106,7 @@
 		.col@{arrayVal} {
 			display: block;
 			float: left;
-			padding: 0 15px;
+			padding: 0 @grid-sides;
 			@media @mq-sm, @mq-md {
 				width: 100;
```
Comment (wylie) [inline less/_grid.less]: 100 what?

File: less/_grid.less [hunk 2]
```diff
@@ -76,9 +76,13 @@
 
 // other CSS for the container of the new columns
 .grid {
+	@grid-sides: 15px;
 	.clearfix;
 
-	width: 100%;
+	padding: 0 25px 40px 25px;
```
Comment (gjjones) [inline less/_grid.less:82]: Should the grid itself have the bottom padding, or should that be something that requires explicit overrides?

**Comment by gjjones** [comment]
What am I missing that this doesn't work?
http://codepen.io/gjjones/pen/zxyqqx

**Comment by gjjones** [comment]
Going to at least `box-sizing: padding-box;` fixed the issue for me with the example markup in the kss. The added padding on the columns throws off the width otherwise.

**Comment by gjjones** [comment]
I'm also not seeing a need for the `margin` and `padding` on the grid. Bootstrap used negative margin well on grids to get initial column flush against the edges, but we don't seem to use it like that.

File: less/_grid.less (hunk 2 above)
Comment (wylie) [inline less/_grid.less:82]: The grid has the padding in order to give some space between rows. I may need to rework this though so we don't have any when we only have one row.

The padding in the side might be an issue though. We have padding on the sides of our interfaces, but I might want to take another look at this.

**Comment by wylie** [comment]
If you add `box-sizing: border-box;` to both the `.grid` and `.col#` classes it should work fine.

**Comment by wylie** [comment]
The negative margins on the sides of the `.grid` class offsets the padding on the `.col#` class. Without that we would have extra padding on the sides of our interfaces.

File: less/_grid.less (hunk 2 above)
Comment (wylie) [inline less/_grid.less:82]: I am having some thoughts on this padding in the `.grid`  I'll see if I can't work them out when I get to the office.

Comment (gjjones) [inline less/_grid.less:82]: That's an interesting point. I get the impression that this class is supposed to represent just rows, so maybe we should make the class `grid-row`?

**Comment by gjjones** [comment]
Ok, so in getting ready to hit the big green "Comment" button after typing out a bunch of stuff, I hit the big green "Merge" button instead.

**Comment by wylie** [comment]
Wait, I wasn't finished…

**Comment by gjjones** [comment]
I wasn't either. Give me a few to revert this back and get the PR back up.

**Comment by wylie** [comment]
No worries, I've got more work to do, but having this in develop shouldn't hurt things. #fat-thumbs

**Comment by gjjones** [comment]
Taking this to flowdock.

**Pull request merged**